### PR TITLE
[maintenance] Various improvements on logs to assist debugging

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Small fixes and improvements while debugging #1752 ...

### e2e
- timeout logs to avoid e2e hanging while creating docker logs
- logs print time-stamps

### webserver
- Adds app settings  [add pydantic (will replace traferet in a new PR) ]
- More metadata: passes build and git commit versions
- Public metadata published via static.json to the front-end and in v0/config entrypoint
- Settings logged in console for reference purposes
- cleanup

### sidecar
- cleanup

### requirements
- weekly update of package requirements
- upgrades to fix vulnerability https://github.com/advisories/GHSA-p9pc-299p-vxgp

## Related issue number

#1752 

## Checklist

- [x] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: packages/pytest-simcore/src/pytest_simcore/postgres_service.py [hunk 1]
```diff
@@ -58,27 +58,41 @@ def postgres_engine(
...
+    # FIXME: migration downgrade fails to remove User types SEE https://github.com/ITISFoundation/osparc-simcore/issues/1776
+    # Added drop_all as tmp fix
     metadata.drop_all(postgres_engine)
 
 
 @pytest.fixture(scope="module")
 def postgres_session(postgres_db: sa.engine.Engine) -> sa.orm.session.Session:
-    Session = sessionmaker(postgres_db)
-    session = Session()
+    from sqlalchemy.orm.session import Session
```
Comment (GitHK) [inline packages/pytest-simcore/src/pytest_simcore/postgres_service.py:88]: is there a reason why this is imported here, or why `sa.orm.session.Session` was not used as a type?

File: services/sidecar/src/simcore_service_sidecar/config.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 )
 
 SIDECAR_LOGLEVEL: str = getattr(
-    logging, os.environ.get("SIDECAR_LOGLEVEL", "WARNING").upper(), logging.WARNING
+    logging, os.environ.get("SIDECAR_LOGLEVEL", "WARNING").upper(), logging.DEBUG
```
Comment (GitHK) [inline services/sidecar/src/simcore_service_sidecar/config.py:46]: 👍

File: services/web/server/src/simcore_service_webserver/projects/projects_db.py [hunk 3]
```diff
@@ -249,13 +253,15 @@ async def load_user_projects(self, user_id: int) -> List[Dict]:
...
             user_groups: List[RowProxy] = await self.__load_user_groups(conn, user_id)
-            query = f"""
-    SELECT *
-    FROM projects
-    WHERE projects.type != 'TEMPLATE'
-    AND (jsonb_exists_any(projects.access_rights, array[{', '.join(f"'{group.gid}'" for group in user_groups)}])
-    OR prj_owner = {user_id})
-    """
+            query = textwrap.dedent(
+                f"""\
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/projects/projects_db.py:257]: just curious, what is the usage of `\` for?

**Review by GitHK — APPROVED** [review]
looks good, just a few questions, personal doubts.

File: packages/pytest-simcore/src/pytest_simcore/postgres_service.py (hunk 1 above)
Comment (pcrespov) [reply packages/pytest-simcore/src/pytest_simcore/postgres_service.py:88]: We agreed not to use sqlalchemy ORMs (aiopg uses a lower level).
 This is an old fixture that we should replace and I put the import inside to keep it all together until we remove it...

File: services/web/server/src/simcore_service_webserver/projects/projects_db.py (hunk 3 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/projects/projects_db.py:257]: Avoids new-line at the beginning:
```cmd
>>> from textwrap import dedent
>>> A = dedent("""
one
   two
   """
)
>>> B = dedent("""\
one
   two
   """
)
>>> A
'\none\n   two\n'
>>> B
'one\n   two\n'
>>> 
```

File: services/sidecar/src/simcore_service_sidecar/executor.py [hunk 4]
```diff
@@ -117,7 +117,9 @@ async def preprocess(self):
         self.shared_folders.create()
         host_name = config.SIDECAR_HOST_HOSTNAME_PATH.read_text()
         await self._post_messages(LogType.LOG, f"[sidecar]Running on {host_name}")
-        results = await logged_gather(self._process_task_inputs(), self._pull_image())
+        results = await logged_gather(
+            self._process_task_inputs(), self._pull_image(), reraise=True
```
Comment (sanderegg) [inline services/sidecar/src/simcore_service_sidecar/executor.py:121]: this is the default.

File: services/web/server/src/simcore_service_webserver/computation_api.py [hunk 5]
```diff
@@ -448,3 +448,4 @@ async def get_task_output(
         comp_task = await result.fetchone()
         if comp_task:
             return comp_task.outputs
+        return None
```
Comment (sanderegg) [inline services/web/server/src/simcore_service_webserver/computation_api.py:451]: well not sure I get the use of adding this. the return type is already marked as Optional

**Review by sanderegg — APPROVED** [review]
some minor comments but that's it. :+1:

File: services/sidecar/src/simcore_service_sidecar/executor.py (hunk 4 above)
Comment (pcrespov) [reply services/sidecar/src/simcore_service_sidecar/executor.py:121]: yes, I just wanted to be more explicit

File: services/web/server/src/simcore_service_webserver/computation_api.py (hunk 5 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/computation_api.py]: you are totally right

**Comment by odeimaiz** [comment]
"log-timestamp" library was NOT installed in the e2e environment, therefore it has been failing since it came in...

If necessary, it should be added to the puppeteer base image

**Pull request merged**